extensions/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+
+val javadocConfig by configurations.creating {
+    extendsFrom(configurations.testImplementation.get())
+}
+
+dependencies {
+    implementation(project(":core"))
+
+    // Use JUnit test framework.
+    testImplementation(libs.junit)
+
+    implementation("com.google.protobuf:protobuf-java:4.27.2")
+    implementation("com.google.flatbuffers:flatbuffers-java:24.3.25")
+}
+
+val isReleaseVersion = !(version as String).endsWith("SNAPSHOT")
+
+tasks.register<Jar>("sourcesJar") {
+    archiveClassifier.set("sources")
+    // See https://discuss.gradle.org/t/why-subproject-sourceset-dirs-project-sourceset-dirs/7376/5
+    // Without the closure, parent sources are used for children too
+    from(sourceSets.getByName("main").java.srcDirs)
+}
+
+mavenPublishing {
+    coordinates(
+        groupId = "io.lionweb.lionweb-java",
+        artifactId = "lionweb-java-${specsVersion}-" + project.name,
+        version = project.version as String,
+    )
+
+    pom {
+        name.set("lionweb-java-" + project.name)
+        description.set("Java APIs for the LionWeb system")
+        version = project.version as String
+        packaging = "jar"
+        url.set("https://github.com/LionWeb-io/lionweb-java")
+
+        scm {
+            connection.set("scm:git:https://github.com/LionWeb-io/lionweb-java.git")
+            developerConnection.set("scm:git:git@github.com:LionWeb-io/lionweb-java.git")
+            url.set("https://github.com/LionWeb-io/lionweb-java.git")
+        }
+
+        licenses {
+            license {
+                name.set("Apache Licenve V2.0")
+                url.set("https://www.apache.org/licenses/LICENSE-2.0")
+                distribution.set("repo")
+            }
+        }
+
+        // The developers entry is strictly required by Maven Central
+        developers {
+            developer {
+                id.set("ftomassetti")
+                name.set("Federico Tomassetti")
+                email.set("federico@strumenta.com")
+            }
+            developer {
+                id.set("dslmeinte")
+                name.set("Meinte Boersma")
+                email.set("meinte.boersma@gmail.com")
+            }
+            developer {
+                id.set("enikao")
+                name.set("Niko Stotz")
+                email.set("github-public@nikostotz.de")
+            }
+        }
+    }
+    publishToMavenCentral(SonatypeHost.S01, true)
+    signAllPublications()
+}
+
+java {
+    sourceCompatibility = JavaVersion.toVersion(jvmVersion)
+    targetCompatibility = JavaVersion.toVersion(jvmVersion)
+}
+
+tasks.jacocoTestReport {
+    dependsOn(tasks.test) // tests are required to run before generating the report
+}
+
+protobuf {
+    protoc {
+        protoc {
+            // Apple Silicon processor would look for an unexisting platform-specific variant
+            artifact = "com.google.protobuf:protoc:4.27.2" + if (osdetector.os == "osx") ":osx-x86_64" else ""
```

Comment:
another version

Reply:
I added a versions catalog and put it there